tasks/gene_typing/task_snippy_variants.wdl

```diff
@@ -61,6 +101,13 @@ task snippy_variants {
     File snippy_variants_bam = "~{samplename}/~{samplename}.bam"
     File snippy_variants_bai ="~{samplename}/~{samplename}.bam.bai"
     File snippy_variants_summary = "~{samplename}/~{samplename}.txt"
+    String snippy_variants_num_reads_aligned = read_string("READS_ALIGNED_TO_REFERENCE")
+    File snippy_variants_coverage_tsv = "~{samplename}/~{samplename}_coverage.tsv"
+    String snippy_variants_num_variants = read_string("VARIANTS_TOTAL")
+    File snippy_variants_depth = "~{samplename}/~{samplename}_depth.tsv"
+    String snippy_variants_ref_length = read_string("REFERENCE_LENGTH")
```

Comment:
Oh, if we're outputting this then it makes sense why you're not piping the commands together. Were these reference length outputs asked for?

Reply:
Discussed on 3/1: reference length is outputted at the task level for troubleshooting but not the workflow level